content/1.7/back-office/shop-parameters/contact/stores/add-edit stores.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Add / Edit Stores
+weight: 1
+---
+
+# Add / Edit Store
+
+When entered the add or editing current store the following fields are displayed:
+
+1) Name
+
+Input field for the name of the store. Can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+
+3) Address
+
+Input field for the address of the shop. This field is mandatory and can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+If there is no address input, when trying to save shows the message: "The field address1 is required at least in English (English)."
+
+5) Address (2)
+
+Second input field for the address of the shop. This field can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+
+7) Zip/Postal code
+
+Input field for the zip/postal code of the shop.
+
```

Comment:
An error message is displayed if the postal code is empty or not in the right format, I let you specify this.